roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1425,6 +1425,13 @@ def get_defaults(self, roles, deployment_type):
                                       auth_token_max_seconds=500,
                                       oauth_grant_method='auto')
 
+        if 'haproxy' in roles:
```

Comment:
This role should probably be loadbalancer now with the openshift_haproxy -> openshift_loadbalancer rename.